tests/test_utils.py

```diff
@@ -15,3 +16,101 @@ def get_app_fixtures(app_list):
             fixtures.append(file)
 
     return fixtures
+
+
+class PermissionTestMixin:
+    user_settings = {
+        'competitor': {
+            'verified_email': True,
+            'is_staff': False,
+            'is_active': True
+        },
+        'kricky': {
+            'verified_email': True,
+            'is_staff': True,
+            'is_active': True
+        },
+        'strom': {
+            'verified_email': True,
+            'is_staff': True,
+            'is_active': True
+        }
+    }
+
+    @classmethod
+    def get_basic_fixtures(cls):
```

Comment:
Hmm, stale si myslim ze test fixtures by mali byt oddelene od fixtures ktore pouzivame len aby nasa databaza nevyzerala byt prazdna. Podla mna by sa testy nemali pokazit len pre to ze si povieme ze ideme teraz naplnit nasu databazu datami ktore su realne a nie len testovacia :D

Ale to je mozno len moj nazor :D

Reply:
Ok, nvm, tieto fixtures su pravdepodobne finalne a nebudu sa nikdy menit... nic som nepovedal :D :D

Reply:
No competition fixtures sú podľa mňa finálne. Anyway väčšinu zých ktoré tam sú teraz by som použil ako test. Prípadne si niekto môže dať práci s tým že vytvorí dvoje. Ale nechcelo sa mi to teraz oddelovat